.gitmodules

```diff
@@ -10,6 +10,8 @@
 [submodule "contracts/lib/eigenlayer-contracts"]
 	path = contracts/lib/eigenlayer-contracts
 	url = https://github.com/Layr-Labs/eigenlayer-contracts
+	branch = m2-mainnet-fixes
```

Comment:
It looks smart contract repo may need a more proper versioning name. Anyway, should the 3 repos above pin the version it depends on?

Reply:
They should. I think Jeffrey is going to come up with a scheme for tracking versions in a more principled way